packages/frontend/components/item/component.tsx

```diff
@@ -34,6 +48,12 @@ export const Item = ({ item }: ItemProps) => {
     dispatch(favAction(item.id));
   };
 
+  useEffect(() => {
+    if (order?.id) {
+      router.push(`${Routes.CHECKOUT}?id=${order?.id}`);
+    }
+  }, [order?.id, router]);
+
```

Comment:
This logic can be handled in action. In that case you don't need to use order here